cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -193,6 +192,20 @@ public KafkaBrokerConfigurationBuilder withZookeeper(String clusterName)  {
         return this;
     }
 
+    /**
+     * Configures ZooKeeper migration related flag
+     *
+     * @param isEnabled if ZooKeeper migration is enabled
+     * @return the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withZooKeeperMigration(boolean isEnabled) {
+        printSectionHeader("Zookeeper migration");
+        writer.println("zookeeper.metadata.migration.enable=" + isEnabled);
+        writer.println();
+
+        return this;
+    }
```

Comment:
I think that in the end the boolean parameter could be totally removed and just set the hard-coded `true`.
This method is never used passing `false` as value, because when broker/controller is configured to be out of migration we just not set the flag in the node configuration. I will remove it.